spec/paperclip/has_attached_file_spec.rb

```diff
@@ -123,6 +127,15 @@ def sets_up_media_type_check_validation
       expect(a_class).to have_received(:validates_media_type_spoof_detection)
     end
 
+    def registers_reflections
+      a_class = stub_class
+
+      Paperclip::HasAttachedFile.define_on(a_class,
+        @attachment_name, { validate_media_type: false })
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Redundant curly braces around a hash parameter.